tests/store/api/conftest.py

```diff
@@ -180,6 +180,102 @@ def fixture_rml_store(store: Store, helpers: StoreHelpers) -> Store:
     return store
 
 
+@pytest.fixture(name="case_id_with_single_sample")
+def case_id_with_single_sample():
+    return "screamingtiger"
+
+
+@pytest.fixture(name="case_id_with_multiple_samples")
+def case_id_with_multiple_samples():
+    return "righteouspanda"
+
+
+@pytest.fixture(name="case_id_without_samples")
+def case_id_without_samples():
+    return "confusedtrout"
+
+
+@pytest.fixture(name="sample_id_in_single_case")
+def sample_id_in_single_case():
+    return "ASM1"
+
+
+@pytest.fixture(name="sample_id_in_multiple_cases")
+def sample_id_in_multiple_cases():
+    return "ASM2"
+
+
+@pytest.fixture(name="dummy_store")
+def fixture_dummy_store(
+    case_id_without_samples: str,
+    case_id_with_single_sample: str,
+    case_id_with_multiple_samples: str,
+    sample_id_in_single_case: str,
+    sample_id_in_multiple_cases: str,
+    helpers,
+    store: Store,
+):
+    sample_in_multiple_cases = helpers.add_sample(
```

Comment:
Add typr hints to tests as well as it makes it easier too understand the test conditions